### PR TITLE
improvement_in_about_us_section_in_index

### PR DESCRIPTION
**Pull request by valentinoarballo**
some issues with blank space in about us section fixed

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -9,10 +9,10 @@
 {% block body %}
 
 <!-- cartelera -->
-<div id="cover-page" class="background-fixed h-700 animated fadeInDown">
+<div id="cover-page" class="background-fixed h-700">
     <div  class="ml-4 d-flex">
         <img src="../static/images/Tux_Mono.png" alt="logo de tuxchange" class="m-3">
-        <h1 class="m-3 text-white">TuXchange</h1>
+        <h1 class="m-3 text-white">Tuxchange</h1>
```
Comment (FacundoEsteban-Lemo) [inline templates/index.html]: Revert this please...

**Review by FacundoEsteban-Lemo — COMMENTED** [review]
Works for me!

**Review by FacundoEsteban-Lemo — APPROVED** [review]

**Review by nazabucciarelliITEC — APPROVED** [review]
In my machine, works good.

**Review by LucaPetrocchi — APPROVED** [review]
OK

**Pull request merged**